src/clr-angular/data/datagrid/datagrid-column-toggle-button.ts

```diff
@@ -10,35 +10,20 @@ import { ColumnToggleButtons, ColumnToggleButtonsService } from './providers/col
 @Component({
   selector: 'clr-dg-column-toggle-button',
   template: `
-        <button
+        <button class="btn btn-sm btn-link"
```

Comment:
We are losing the `text-uppercase` class here, is it not needed?

Reply:
Nope, both uppercase an `p6` styles are already part of button styles.